src/DataDictionary/utils.js

```diff
@@ -92,3 +93,100 @@ export const graphStyleConfig = {
   nodeIconRadius: 10,
 };
 
+export const parseDictionaryNodes = dictionary => Object.keys(dictionary).filter(
+  id => id.charAt(0) !== '_' && id === dictionary[id].id,
+).map(
+  (id) => {
+    const originNode = dictionary[id];
+    return originNode;
+  },
+).filter(
+  node => node.category && node.id,
+);
+
+export const getPropertyDescription = (property) => {
+  let description;
+  if ('description' in property) {
+    description = property.description;
+  }
+  if ('term' in property) {
+    description = property.term.description;
+  }
+  return description;
+};
+
+const searchHistoryLocalStorageKey = 'datadictionary:searchHistory';
+/**
+ * @typedef {Object} SearchHistoryItem
+ * @property {string} keywordStr - keywordStr of this item
+ * @property {integer} matchedCount - matched count for this keyword
+ */
+
+/**
+ * Get search history items from localStorage
+ * @returns {SearchHistoryItem[]} array of search history items
+ */
+export const getSearchHistoryItems = () => {
```

Comment:
is it necessary to store search history items in local storage? what about the Redux store?